go-controller/pkg/util/batching/batch.go

```diff
@@ -0,0 +1,25 @@
+package batching
+
+import "fmt"
+
+const DefaultBatchSize = 1000
```

Comment:
nit: maybe we don't need to define a default since we don't really know what the batching utility will be used for. So this could be better defined in egressfirewall.go itself